web/pdf_print_service.js

```diff
@@ -298,6 +298,7 @@ if (hasAttachEvent) {
       event.keyCode = 0;
       return false;
     }
+    return undefined;
```

Comment:
There are more `return` statements without `undefined` above. Why are they not picked up?

Also, shouldn't this return a boolean like we do in the `if` statement?

Reply:
Again, the key is consistent `return`s here; so https://github.com/mozilla/pdf.js/pull/10805#discussion_r283094312 here too :-)

I'm not sure that I really understand this code, but I suppose that there *could* be a difference between returning `false` and `undefined` in an `attachEvent`.
However, I think that I'll just disable that particular one since I don't want to unknowingly cause any regressions here.